packages/entity-browser/src/components/DetailView/LabeledField.js

```diff
@@ -2,6 +2,24 @@ import React from 'react'
 import _clone from 'lodash/clone'
 import * as ToccoUi from 'tocco-ui'
 
+const ErrorList = props => {
+  const errorValues = Object.values(props.errors)
```

Comment:
This leads to an error if there is no `error` prop.

Possibilities:

- Declare prop as required
- Declare default prop `{}`
- Check here if errors are set